doc/fuzzing.md

```diff
@@ -87,11 +102,33 @@ To build all fuzz targets with libFuzzer, run
 make
 ```
 
-The fuzzer needs some inputs to work on, but the inputs or seeds can be used
-interchangeably between libFuzzer and AFL.
-
 See https://llvm.org/docs/LibFuzzer.html#running on how to run the libFuzzer
 instrumented executable.
 
 Alternatively run the script in `./test/fuzz/test_runner.py` and provide it
 with the `${DIR_FUZZ_IN}` created earlier.
+
+### macOS hints for libFuzzer
+
+The default clang/llvm version supplied by Apple on MacOS does not include
+fuzzing libraries, so macOS users will need to install a full version, for
+example using `brew install llvm`.
+
+When executing ./configure you should put --disable-asm to avoid errors
```

Comment:
What errors are you seeing with the asm? Can you post logs?

Reply:
Hm, I am sure I had an issue without it before but I can not reproduce it right now and I did not keep the logs since I tried libFuzzer one week ago. So it is possible that it is not an issue anymore or I simply mistook this as fixing an unrelated issue. However, the issue is also noted in the docs on sanitizers: https://github.com/bitcoin/bitcoin/blob/master/doc/developer-notes.md#sanitizers